docs/crio.8.md

```diff
@@ -107,7 +107,7 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 # GLOBAL OPTIONS
 
-**--additional-devices**="": Devices to add to the containers (default: [])
+**--additional-devices**="": Devices to add to the containers (default: []) (default: [])
```

Comment:
I'm not sure what the double default means here and at line 120?

Reply:
make that through out.

Reply:
Ah I see, let me check if I can fix this...